examples/nemotron-h/README.md

```
@@ -1,23 +1,33 @@
-# Nemotron-H (nvidia/NVIDIA-Nemotron-3-*)
+# Nemotron-H (nvidia/NVIDIA-Nemotron-3-* and NVIDIA-Nemotron-3.5-*)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)